config/default.json

```diff
@@ -6,8 +6,8 @@
   "espace_coop_hostname": "https://beta.coop.conseiller-numerique.gouv.fr",
   "port": 8080,
   "paginate": {
-    "default": 10,
-    "max": 50
+    "default": 20,
```

Comment:
@loicguillois Normal ces changements ?

Reply:
oui on augmente le nombre de lignes affichées

Reply:
Ok noté ;)